.github/workflows/cloudcompiler.yaml

```diff
@@ -0,0 +1,52 @@
+name: Release cloudcompiler.wasm
+
+env:
+  RUST_BACKTRACE: 1
+
+on: 
+  pull_request:
+    branches:
+      - 'master'
+  workflow_dispatch:
+
+jobs:
+  setup:
+    name: Set up
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: 1.59
+          target: ${{ matrix.target }}
+      - name: Install wasm32-wasi target
+        shell: bash
+        run: |
+          rustup target add wasm32-wasi
+      - name: Build cloudcompiler.wasm
+        shell: bash
+        run: |
+          cargo build --release --target wasm32-wasi &&
+          mkdir target/release/cloudcompiler &&
+          cp target/release/wasmer.wasm target/release/cloudcompiler/cloudcompiler.wasm &&
```

Comment:
For me, the file is generated in `target/wasi32-wasm/release` and is name `wasmer-compiler.wasm`

Reply:
yeah this was just a test / initial commit, I need to rework this script anyway